gcp_variant_transforms/libs/annotation/vep/vep_runner.py

```diff
@@ -194,7 +194,7 @@ def _get_api_request_fixed_parts(self):
     }
 
   def _make_action(self, *args, **kwargs):
-    # type: (str, str) -> Dict
+    # type: (*str, **str) -> Dict
```

Comment:
@bashir2 Can you help check this method? There is a warning here. It seems that it is passing a List instead a dictionary.

Reply:
So like we discussed, the correct type is `**List[str]`, thanks for fixing this.

Reply:
Updated. Thanks!